Makefile.am

```diff
@@ -304,6 +304,13 @@ BATS = \
 	test/functional/usability/usa-completion-basic.bats \
 	test/functional/usability/usa-download-retries.bats \
 	test/functional/usability/usa-external-modules.bats
+	test/functional/signature/cert-chain.bats \
+	test/functional/signature/corrupted-certificate.bats \
+	test/functional/signature/invalid-certificate.bats \
+	test/functional/signature/permission-incorrect.bats \
+	test/functional/signature/corrupted-signature.bats \
+	test/functional/signature/no-signature.bats
+
```

Comment:
you missed adding a  '\' at the end of `test/functional/usability/usa-external-modules.bats` so your tests are never gonna run. Also the list of tests is sorted alphabetically, I'd prefer if it stayed that way, it makes easier to find tests in the list.

Reply:
Also you forgot to add this test `test/functional/signature/key-rotation.bats` to the list in Makefile.am